bluebottle/wallposts/admin.py

```diff
@@ -11,20 +14,36 @@
 from bluebottle.wallposts.models import SystemWallpost
 from .models import Wallpost, MediaWallpost, TextWallpost, MediaWallpostPhoto, Reaction
 
+from sorl.thumbnail.shortcuts import get_thumbnail
+
 
-class MediaWallpostPhotoInline(AdminImageMixin, admin.StackedInline):
+class MediaWallpostPhotoInline(admin.TabularInline):
     model = MediaWallpostPhoto
     extra = 0
-    raw_id_fields = ('author', 'editor')
+    raw_id_fields = ('author', 'editor') # for performance reasons?
```

Comment:
Not just for performance, it's also a way-better UX.
@iivvoo could you remove this comment :-)